cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -448,6 +449,10 @@ subroutine init_grid2
 
       !-----------------------------------------------------------------
       ! T-grid cell and U-grid cell quantities
+      ! Fill halo data locally where possible to avoid missing
```

Comment:
Is this change BFB when blocks are not eliminated?

Reply:
I've run the quick suite on cheyenne pgi, intel, and gnu and it is BFB so far.  I was worried about this too.  By avoiding the haloupdate for the variables, we can do a little more in terms of the averaging to/from grids for blocks where the neighbor is eliminated.  I need to test the full suite and will do that tonight.